src/transformers/modeling_t5.py

```diff
@@ -900,8 +900,10 @@ def forward(
         r"""
         lm_labels (:obj:`torch.LongTensor` of shape :obj:`(batch_size,)`, `optional`, defaults to :obj:`None`):
                 Labels for computing the sequence classification/regression loss.
-                Indices should be in :obj:`[0, ..., config.vocab_size - 1]`.
+                Indices should be in :obj:`[-100, 0, ..., config.vocab_size - 1]`.
                 If :obj:`config.num_labels > 1` a classification loss is computed (Cross-Entropy).
```

Comment:
Should we keep this `num_labels` mention in the docstring?

Reply:
True, T5 is completely independent from `num_labels` -> should delete this statement.